internal/core/ports/operation_ports.go

```diff
@@ -88,6 +88,8 @@ type OperationStorage interface {
 	// UpdateOperationStatus only updates the operation status for the given
 	// operation ID.
 	UpdateOperationStatus(ctx context.Context, schedulerName, operationID string, status operation.Status) error
+	// UpdateOperationExecutionHistory updates the operation execution history in operation storage
```

Comment:
The name of the interface is OperationStorage, I don't think we need to mention that it updates in the operation storage here, right?